scripts/get-repos/README.md

```diff
@@ -0,0 +1,30 @@
+# Repo List
+
+List all PL repos. Currently, this script lists all Go & JavaScript repos in the following orgs:
+
+- ipfs
+- ipfs-shipyard
+- ipld
+- libp2p
+- multiformats
+
+It outputs them as a JSON array.
+
+## Install
+
+Before usage, you'll need to install the dependencies.
+
+```go
+$ npm install
+```
+
+## Usage
+
+```bash
+$ node index.js
+```
+
+If you run into GitHub's rate limits, try setting the GITHUB_TOKEN environment variable:
+```bash
+$ GITHUB_TOKEN=<your token> node index.js
```

Comment:
nit: maybe `${your_token}` or just `your_token` would be a bit nicer syntax, since `<` and `>` are redirects